docs/source/configuration/volto-slate/configuration-settings.md

```diff
@@ -239,6 +239,15 @@ They are not persisted in the final value, so they are useful, for example, to h
 slate.runtimeDecorators = [([node, path], ranges) => ranges];
 ```
 
+(editor-configuration-slate-useLinkedHeadlines-label)=
```

Comment:
We need to be consistent: heading != headline. A headline is usually a title, with some headings below. This might mean you would need to refactor the code as well.

```suggestion
(editor-configuration-slate-useLinkedHeading-label)=
```

Reply:
[de9e57f](https://github.com/plone/volto/pull/4287/commits/de9e57f3797e705a451351046a9688722a28500c)